### ISSUE TITLE
Chart resizes when it get's new data via setting results

### ISSUE THREAD

**Issue by Razzeee** [swimlane/ngx-charts#783]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
I'm testing a line chart, but I would think this is also a problem with others. I'm not using a `view`-option, thus the chart should redraw on window changes.

If I load the site everything is fine. When I then get new data and refresh the data inside `results` height changes. It seems to want to endup somewhere around the 400px mark.

Here a example snippet from the html.
```
<div class="card-body m-2">
  <ngx-charts-line-chart [results]="chartData" [xAxis]="true" [yAxis]="true"
    [showYAxisLabel]="true" [yAxisLabel]="yAxisLabel">
  </ngx-charts-line-chart>
</div>
```

**Expected behavior**

I would expect the component to not resize, or at least resize and still honor the window it is in.

**Reproduction of the problem**

Wasn't able to get a plnkr going that also has this problem. :(

**What is the motivation / use case for changing the behavior?**
Well I don't want ngx-charts to kill my layout :)

**Please tell us about your environment:**
Windows + NPM

* **ngx-charts version:** 7.1.1

* **Angular version:** 5.2.4

* **Browser:** Tested current chrome and firefox bot have this.

* **Language:** Tested with Typescript

**Comment by Razzeee** [comment]
Btw if I set `view` everything is fine, but I want it to resize automatically or at least resize the width automatically

**Comment by devakone** [comment]
If by default , and by that I mean you do not specify a `view` option, the behavior is to fill the size of the chart container. Why do you set a `max-height` on the container to a value you're comfortable with?

**Comment by Razzeee** [comment]
Which container are you referencing? The `div` for the card-body? If I set `height` only to something like 300px the graph will still grow after first refresh. Ending in it drawing over the containers bounds.
![image](https://user-images.githubusercontent.com/5943908/38294813-71cd095e-37ec-11e8-9676-e15f2fff8111.png)

If I only set `max-height` it will still grow after first refresh, but respect that size and still look fine.
So I ended up setting both `max-height` and `height` for now. Which seems to solve it.
